src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/unusedimports/InputUnusedImportWithBlockMethodParameters.java

```diff
@@ -0,0 +1,15 @@
+package com.puppycrawl.tools.checkstyle.checks.imports.unusedimports;
+
+import java.util.Collection;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.TimeUnit;
+
+public class InputUnusedImportWithBlockMethodParameters {
+
+/**
+ * @see ExecutorService#invokeAll(Collection, long, TimeUnit)
```

Comment:
Ok, thanks. 

I'm done with this PR. The build is failing because issue #4606 isn't approved, yet. 

Let me know if there's anything I can do to get the issue approved and this PR merged.